WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustoms.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import WooFoundation
+
+struct WooShippingCustoms: View {
+    let informationIsCompleted: Bool
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        HStack {
+            Text(Localization.customsTitle)
+                .font(.system(size: Layout.customsTitleFontSize, weight: .medium))
```

Comment:
Do we need to specify a fixed font size here? It's preventing dynamic font resizing:

Regular size|Increased size
-|-
![Simulator Screenshot - iPhone 16 Pro - 2024-12-19 at 12 10 21](https://github.com/user-attachments/assets/03cc28d0-d63e-4b76-bed5-054397790541)|![Simulator Screenshot - iPhone 16 Pro - 2024-12-19 at 12 10 26](https://github.com/user-attachments/assets/f549f219-8746-483b-8e9d-c1889aa738a2)

I'd expect that we could use the `.headlineStyle()` modifier to get the same effect while still supporting dynamic fonts.

Reply:
Good point, done in https://github.com/woocommerce/woocommerce-ios/pull/14729/commits/8130522b485ded46f5c545185fec3ec253951c42